src/scss/_pseudo-radio-link.scss

```diff
@@ -4,9 +4,8 @@
 @mixin _oFormsPseudoRadioLink {
 	.o-forms-input--pseudo-radio-link {
 		@include _oFormsControlsBoxContainer();
-		@include oTypographyMargin($top: 0);
-
 		@include _oFormsPseudoRadioLinkStyles();
+		margin-top: 0;
```

Comment:
are these the same?!

Reply:
Yup! [oTypographyMargin](https://github.com/Financial-Times/o-typography/blob/1725d765269f32cb0b077004e44c7df1e20da9a7/src/scss/_mixins.scss#L37) outputs the long form syntax for top and or bottom margins. With just one `$top` argument it just outputs the top margin. 

Also `_oFormsPseudoRadioLinkStyles` doesn't output a margin property, so the re-order is ok.

Reply:
!!!